README.md

```diff
@@ -102,11 +102,13 @@ Each time you need to consume recent EDS changes elsewhere, you need to bump the
 - Prompt you for the new version number
 - Update `package.json` with the version number
 - Commit the change in a new commit
+- Get your PR approved in your branch and merge it to master
 
-2. To create and push a git tag, do one of these:
+2. To create and push a git tag:
```

Comment:
This should just say `To push a git tag:`  


[IIUC docs](https://classic.yarnpkg.com/en/docs/cli/version#toc-git-tags) the tag is being created when you run `yarn version`.
> If you run yarn version within a Git repository an [annotated Git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) will be created by default following the format v0.0.0.